tests/www/test_views.py

```diff
@@ -1154,6 +1154,11 @@ def test_page_instance_name_xss_prevention(self):
 
 
 class TestConfigurationView(TestBase):
+    def setUp(self):
+        super().setUp()
+        with mock.patch.dict(os.environ, {"AIRFLOW__CORE__UNIT_TEST_MODE": "False"}):
+            initialize_config()
```

Comment:
Is this needed?

Reply:
Yes. Otherwise one of the WWW tests fail when WWW is run in full isolation. This test is written with the assumption that the config is created but if it is run in isolation, the config does not exist. This is fix to those tests - making sure that those tests do not rely on side effect from other tests (this configuration file is written by other tests that's why it has never been caught before.

Reply:
@potiuk Can you explain why you are setting unit test mode to false here? At first glance that seems incorrect.

Reply:
Sure. This tests reads value of the airflow configuration and returns it to the caller. The test failed (when I run it in parallel) because the config was missing. The web api returns the "production" config - no matter if the UNIT_TEST_MODE is set to True or not. I have not yet had time to figure out why the "airflow.cfg" is missing when I run the test in parallel mode (it is there when we run tests sequentially). 

I still have to figure it out (and planned to). My guess is that some factor influences the sequence of executing the tests and some other test simply deletes the configuration as part of its execution and the config never gets restored when the `test_configuration_expose_config' is executed. Unfortunately presence of config is 'implicit dependency' in many of our tests - the problem is that the config gets automatically created by airflow configuration code when it is missing, so if any test depend on its presence it might went unnoticed that the config is actually a side effect of either other tests or simply running airflow cli. 

So I believe my fix is `proper` fix (but there are probably other tests that have similar side-effects dependency). Since this test depends on the 'airflow.cfg` being present, this should not be side-efffect, it should be explicitly initialised int 'setUp' of the test. Which is precisely what I do - I make sure that the "unit_test_mode" is set to false, so the "production" and not "unittest" config is initialized and then I Initialise it.

I think this is how the test should be written in the first place.

Reply:
I was thinking about separating out this into separate PR, which I will likely do in the evening when I switch to my personal PC.